Ports/zlib/package.sh

```diff
@@ -5,12 +5,6 @@ useconfigure=true
 files="https://www.zlib.net/zlib-${version}.tar.gz zlib-${version}.tar.gz b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30"
 
 configure() {
-    # Set uname to linux to prevent it finding the host's `libtool` on e.g. Darwin
-    run ./configure --uname=linux
-}
-
-install() {
-    run make DESTDIR=${SERENITY_INSTALL_ROOT} "${installopts[@]}" install
-    ${CC} -shared -o ${SERENITY_INSTALL_ROOT}/usr/local/lib/libz.so -Wl,-soname,libz.so -Wl,--whole-archive ${SERENITY_INSTALL_ROOT}/usr/local/lib/libz.a -Wl,--no-whole-archive
-    rm -f ${SERENITY_INSTALL_ROOT}/usr/local/lib/libz.la
+    export LDSHARED="$CC -shared -Wl,-soname,libz.so"
```

Comment:
How come that we have to specify this manually here? I assume that's what you meant by "tell its (non-autotools) configure script how to link a shared library for SerenityOS", but I'd assume that at least the soname would be done by zlib internally.

Reply:
`configure` doesn't set the soname on unknown platforms, see line 270.

Reply:
I added a short comment to explain this.